test/watch/watch-flag.test.js

```diff
@@ -1,16 +1,20 @@
 'use strict';
 
-const { runWatch, isWindows } = require('../utils/test-utils');
+const { runWatch, isWindows, runAndGetWatchProc } = require('../utils/test-utils');
+const { writeFileSync } = require('fs');
+const { resolve } = require('path');
+const { version } = require('webpack');
 
-describe('--watch flag', () => {
-    if (isWindows) {
-        it('TODO: Fix on windows', () => {
-            expect(true).toBe(true);
-        });
-        return;
-    }
+const wordsInStatsv4 = ['Hash', 'Version', 'Time', 'Built at:', 'main.js'];
+const wordsInStatsv5 = ['asset', 'index.js', 'compiled successfully'];
 
+describe('--watch flag', () => {
     it('should watch for file changes', async () => {
+        if (isWindows) {
+            // TODO: Fix on windows
+            expect(true).toBe(true);
+            return;
+        }
```

Comment:
still not working on windows?

Reply:
I don't know for what reason old test was disabled on windows, so I left it the way it was.

But with new test, which runs on windows, I don't think we need this one now. Will remove old one.